detox/test/detox.config.js

```diff
@@ -0,0 +1,150 @@
+/** @type {DetoxConfig} */
+module.exports = {
```

Comment:
Shouldn't we remove this from `package.json`, in that case?

Reply:
Yep, forgot. 😛